packages/web/components/table/assets-table-v1.tsx

```diff
@@ -752,3 +792,27 @@ export const AssetsTableV1: FunctionComponent<Props> = observer(
     );
   }
 );
+
+const useWebSearchBoxState = () => {
+  const [searchBoxIsFocused, setSearchBoxIsFocused] = useControllableState({
+    defaultValue: false,
+  });
+  const searchBoxRef = useRef<HTMLInputElement>(null);
+  useKey(
+    "/",
+    (event) => {
+      event.preventDefault(); // Prevent the '/' from being entered into the input
+      searchBoxRef.current?.focus();
+    },
+    { event: "keydown" },
+    [searchBoxRef]
+  );
+  useKey("Escape", () => searchBoxRef.current?.blur(), { event: "keydown" }, [
+    searchBoxRef,
+  ]);
```

Comment:
Since refs are stable, we can skip adding them to the dependency array 

```suggestion
  useKey(
    "/",
    (event) => {
      event.preventDefault(); // Prevent the '/' from being entered into the input
      searchBoxRef.current?.focus();
    },
    { event: "keydown" },
    []
  );
  useKey("Escape", () => searchBoxRef.current?.blur(), { event: "keydown" }, []);
```